lib/searchable_model.rb

```diff
@@ -38,28 +38,29 @@ def search(search, page, user, includes={})
     # pass in a username to limit the search to the users items
     if (!user.nil?) && (!user.id.nil?)
       if column_names.include? 'user_id'
-        if self == User
-          sql_conditions = ""
-        else
+        if self != User
```

Comment:
I guess if this code is working then the users table doesn't have a id called `user_id`. Otherwise the current_user would only be able to see their own user in the search results.